apps/public-docsite/src/pages/Styles/FabricIconsPage/docs/web/FabricIconsUsage.md

```diff
@@ -41,6 +41,20 @@ const MyIconButton = () => <IconButton iconProps={{ iconName: 'Add' }} title="Ad
 ReactDOM.render(<MyIconButton />, document.body.firstChild);
 ```
 
+### New SVG Based Fluent UI Icons
```

Comment:
You should probably change the "Fluent UI React" heading (earlier) to say "Fluent UI React: font-based icons" and this one to say "Fluent UI React: SVG-based icons". Either that or add level 4 headings "Font-based icons" and "SVG-based icons" under the level 3 "Fluent UI React" heading. (Also, headings are sentence case not title case.)